requirements.txt

```diff
@@ -5,11 +5,11 @@ djangorestframework-simplejwt==5.2.0
 django-cors-headers==3.13.0
 pyjwt==2.4.0
 # Algorithms versions should be updated regularly
-coursescheduler==0.0.1 #Company 2 alg 1 
-c1algo1==1.0.11 # Company 1 alg 1
+coursescheduler #Company 2 alg 1
+c1algo1 # Company 1 alg 1
 
 --extra-index-url https://test.pypi.org/simple/
-capacityforecaster==0.1.5 #Company 2 alg 2 (from pypi.org/simple/ index)
+capacityforecaster #Company 2 alg 2 (from pypi.org/simple/ index)
```

Comment:
Removed version numbers from algs packages so we're just always working with their latest release